base/int.jl

```diff
@@ -332,6 +332,21 @@ for to in BitInteger_types, from in (BitInteger_types..., Bool)
     end
 end
 
+"""
+    rem(x::Integer, T<:Integer)
+    mod(x::Integer, T<:Integer)
+    %(x::Integer, T<:Integer)
+
+Find `y::T` such that `x` ≡ `y` (mod n), where n is the number of integers representable
+in `T`, and `y` is an integer in `[typemin(T),typemax(T)]`.
```

Comment:
Changed `by -> in` on purpose. Felt like it sounded more natural. "Represented in" suggests a collection. "Represented by" suggests a mapping (like lie algebra element -> matrix operator). I thought `Int8` etc. is more like a collection than like a mapping. Is there a convention on this in the docs? Happy to change back